### PR TITLE
test coverage for Get-Service

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Part of https://github.com/PowerShell/PowerShell/issues/4147

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1 [hunk 1]
```diff
@@ -40,4 +41,51 @@
...
     }
   }
+
+  It "GetServiceCommand can be used as API for '<parameter>' with '<value>'" -TestCases @(
+    @{parameter="DisplayName";value="foo"},
+    @{parameter="Include";value="foo","bar"},
+    @{parameter="Exclude";value="bar","foo"},
+    @{parameter="InputObject";script={Get-Service | Select-Object -First 1}},
+    @{parameter="Name";value="foo","bar"},
+    @{parameter="ComputerName";value="bar","foo"}
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1]: Maybe reformat? And below too?
```powershell
    @{parameter="DisplayName";  value="foo"},  
    @{parameter="Include";      value="foo","bar"},  
    @{parameter="Exclude";      value="bar","foo"},  
    @{parameter="InputObject";  script={Get-Service | Select-Object -First 1}},  
    @{parameter="Name";         value="foo","bar"},  
    @{parameter="ComputerName"; value="bar","foo"}  
```

Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1]: Will update

File: test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1 [hunk 2]
```diff
@@ -40,4 +41,51 @@
...
+    if ($servicesCheck -ne $null) {
+      Compare-object $services $servicesCheck | Out-String | Should BeNullOrEmpty
+    } else {
+      $services | Should BeNullOrEmpty
+    }
+  }
+
+  It "Get-Service fails for non-existing service using '<script>'" -TestCases @(
+    @{script={Get-Service -Name (new-guid) -ErrorAction Stop};ErrorId="NoServiceFoundForGivenName,Microsoft.PowerShell.Commands.GetServiceCommand"},
+    @{script={Get-Service -DisplayName (new-guid) -ErrorAction Stop};ErrorId="NoServiceFoundForGivenDisplayName,Microsoft.PowerShell.Commands.GetServiceCommand"}
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1]: Could you please indent ErrorId too?

**Comment by iSazonov** [comment]
@SteveL-MSFT Is the PR ready to merge?

File: test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1 (hunk 2 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1]: sure

**Comment by SteveL-MSFT** [comment]
@iSazonov take a look again, if you can approve, just give it a day and then merge

**Review by iSazonov — APPROVED** [review]
Leave a comment

**Pull request merged**